examples/plot_intelex.py

```diff
@@ -120,7 +120,7 @@
 log_loss_score = log_loss(y_test, y_proba)
 print(
     f"[stock scikit-learn] Inference took t_stock = {t_stock:.2f}s with a "
-    f"log-loss score of {log_loss_score:.1%}"
+    f"log-loss score of {log_loss_score:.1f}"
```

Comment:
```suggestion
    f"log-loss score of {log_loss_score:.3f}"
```

Let's put 3 digits, otherwise, it's just `0.2`.